packages/components/src/components/ContentCard/ContentCard.tsx

```diff
@@ -103,28 +91,43 @@ export const ContentCard = ({
     },
   };
 
+  const isCardInteractive = interactiveElementType === "card";
+  const isButton = interactiveElementType === "button";
+  const inAnchor = interactiveElementType === "anchor";
```

Comment:
Maybe having `interractive` in all variables here?
Another thing I think would be good is using an enum. So you could use its value instead of strings. 

```typescript
isInteractiveElementType(InteractiveElementType.BUTTON);
```